howto/container/launch.md

```diff
@@ -53,9 +53,9 @@ By default, every container is scheduled by AMS onto a LXD node. Alternatively,
 
 ## Launch a container with a different Anbox platform
 
-By default, every container starts with the `null` platform (see [Anbox platforms](https://discourse.ubuntu.com/t/anbox-platforms/18733)). The selected platform cannot be changed at runtime and must be selected when the container is created. For example, you can launch a container with the `swrast` platform like this:
+By default, every container starts with the `null` platform (see [Anbox platforms](https://discourse.ubuntu.com/t/anbox-platforms/18733)) and with the `webrtc` platform when `--enable-graphics` is specified. The selected platform cannot be changed at runtime and must be selected when the container is created. For example, you can launch a container with the `webrtc` platform like this:
```

Comment:
@ru-fu this reads a bit weird: `by default, [...] if --enable-graphics is specified, otherwise [...]`
Shouldn't the `By default` be placed after ?

Reply:
Maybe it's best to just leave out the "by default". Originally it was there to say "if the platform is not specified" - but now it depends on both `-p` and `--enable-graphics`.

```suggestion
Containers start with the `webrtc` platform if `--enable-graphics` is specified and with the `null` platform otherwise (see [Anbox platforms](https://discourse.ubuntu.com/t/anbox-platforms/18733)). To select a different platform, specify it with the `-p` flag. The selected platform cannot be changed at runtime and must be selected when the container is created. For example, you can launch a container with the `webrtc` platform like this:
```

Reply:
Actually, what happens if you specify `--enable-graphics -p null`?

Reply:
> Maybe it's best to just leave out the "by default"
Works fine for me

> Actually, what happens if you specify `--enable-graphics -p null`?

You can't launch the container with both options enabled, the command will return an error if you do